config/crd/kustomization.yaml

```
@@ -6,7 +6,7 @@ resources:
 - bases/infrastructure.cluster.x-k8s.io_dockerclusters.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
-patches:
+#patches:
```

```suggestion
patchesStrategicMerge: []
```

let's update it and keep it around. `patches` is simply invalid in kustomize v3.x